reqs/base.txt

```diff
@@ -1,3 +1,4 @@
+croniter==0.3.29
```

Comment:
How is this relevant for this change?

Reply:
Tests don't run without installing this library